build.gradle

```diff
@@ -2396,13 +2431,14 @@ project(":swing") {
         //shims // no test shims needed
         test
     }
-
+            
     project.ext.moduleSourcePath = defaultModuleSourcePath
     project.ext.moduleSourcePathShim = defaultModuleSourcePathShim
 
     commonModuleSetup(project, [ 'base', 'graphics', 'swing' ])
 
     dependencies {
+        compile project(":media")
```

Comment:
`javafx.swing` does not depend on `javafx.media`. I think this should instead be:

```
                compile project(":base")
                compile project(":graphics")
```